### DISCUSSION TITLE
How to update language file django.po from crowdin.com

### DISCUSSION THREAD

**Discussion by betoxx** [inventree/InvenTree#2490] [Q&A] [answered]
Hi!! I was contributing translating into Spanish in crowdin.com, and my question is where do I must upload the file django.po once downloaded to update Inventree?

**Comment by matmair** [comment, marked as answer]
@betoxx thanks for the effort! After the translations are approved they get regularly merged into the main branch and will be shipped with the next release.

**Reply by matmair** [reply]
@betoxx please mark this as the answer if this fulfills your question.

**Reply by betoxx** [reply]
Only one question, after you merge it into the main branch, what is the procedure to synchronize the changes without doing all the install process from the beginning?

**Reply by matmair** [reply]
You run the normal update procedure: pull updates from github and run the update routine:
https://inventree.readthedocs.io/en/latest/start/update/

It is detailed in the update section

**Reply by matmair** [reply]
The strings often change so it does not really make sense to distribute them separately as we would need to version them quite rigerously.
You can however just pull the .po that you want and place it into the /locales/ folder of the desired language and run
Invoke translate
From a terminal within inventrees venv enviroment.
